prov/shm/configure.m4

```diff
@@ -14,9 +14,12 @@ AC_DEFUN([FI_SHM_CONFIGURE],[
 	AS_IF([test x"$enable_shm" != x"no"],
 	      [
 	       # check if CMA support are present
-	       AC_CHECK_FUNC([process_vm_readv],
-			     [cma_happy=1],
-			     [cma_happy=0])
+	       AS_IF([test x$linux = x1 && test $host_cpu = xx86_64],
```

Comment:
You need an 'x' before $host_cpu for this check

Reply:
Oh yup, sorry my bad. Updated.